tests/rados/test_osd_db_wal_configuration.py

```diff
@@ -0,0 +1,207 @@
+                        f"{cmd_base} ceph-volume lvm new-wal --osd-id {test_osd} --osd-fsid {osd_fsid} "
+                        f"--target LVMTest/new_wal"
+                    )
+                    log.info(f"Configuring the new wal for the {test_osd} osd ")
+                else:
+                    device_flag = "db"
+                    cmd_add_db_wal = (
+                        f"{cmd_base} ceph-volume lvm new-db --osd-id {test_osd} --osd-fsid {osd_fsid} "
+                        f"--target LVMTest/new_db"
+                    )
+                    log.info(f"Configuring the new db for the {test_osd} osd ")
+                conf_output, _ = osd_node.exec_command(sudo=True, cmd=cmd_add_db_wal)
+
+                log.info(f"The configuration out out is- {conf_output}")
+
+                after_configure_lvm = rados_object.get_ceph_volume_lvm_list(
+                    osd_node, test_osd
+                )
+                log.info(
+                    f" ceph volume lvm list for the osd{test_osd} after configuration is {after_configure_lvm}"
+                )
+                # start test osd
+                if not rados_object.change_osd_state(action="start", target=test_osd):
+                    log.error(f"Unable to stop the OSD : {test_osd}")
+                    raise Exception("Execution error")
+                tc_verification_result = result_verification(
+                    before_configure_lvm, after_configure_lvm, test_osd, device_flag
+                )
+                if tc_verification_result is False:
+                    log.error(
+                        f"{device_flag} configuration failed on {test_osd} in the {osd_node.hostname}"
+                    )
+                    return 1
+        if test_count == 0:
+            log.error(" Configuration failed on all the nodes")
+            return 1
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    return 0
+
+
+def count_available_SSD(total_list):
+    """
+    Method return the number of available SSD's in the cluster
+    Args:
+        total_list: Total device list in the json format
+    Returns:Return the number of SSD's in the cluster
+
+    """
+    count = 0
+    for device_list in total_list:
+        for devices in device_list["devices"]:
+            if devices["available"] is True and devices["human_readable_type"] == "ssd":
+                count += 1
+    return count
+
+
+def get_ssd_path(total_list):
+    """
+    Method return the available SSD paths in the cluster
+    Args:
+        total_list: Total device list in the json format
+    Returns: Method returns a dictionary with node name as key and SSD path list as values
+
+    """
+    ssd_paths = {}
+    path = []
+    for device_list in total_list:
+        path.clear()
+        for devices in device_list["devices"]:
+            if devices["available"] and devices["human_readable_type"] == "ssd":
```

Comment:
Are Nvme drives also identified as SSD?

Reply:
Yes, SSD and NVME drivers are under the SSD section.

Sample Json:
[{"addr": "mero006", "devices": [{"available": false, "ceph_device": true, "created": "2023-11-10T10:17:49.212707Z", "crush_device_class": null, "device_id": "MTFDHBG960MCH-1AN1ZABYY_ZF1000XW", "**human_readable_type": "ssd"**, "lsm_data": {}, "lvs": [{"cluster_fsid": "d111a452-7fac-11ee-b783-ac1f6b40d3fc", "cluster_name": "ceph", "name": "new_wal", "osd_fsid": "b70c7e8d-f572-4393-865b-8b953639968a", "osd_id": "0", "osdspec_affinity": "osd_spec_hdd", "type": "wal", "wal_uuid": "PTtnHC-ZzLb-9A9l-Y7y6-rp48-f4y0-2HrIYV"}], "**path": "/dev/nvme0n1",**